webpack_config/main.js

```diff
@@ -5,7 +5,7 @@ const { merge } = require('webpack-merge');
 const common = require('./common');
 
 module.exports = merge(common, {
-  entry: './src/main.ts',
+  entry: { index: './src/main.ts', worker: './src/signing/signing.worker.ts' },
```

Comment:
We may need a separate Webpack config for the worker, to make sure it does not include any unnecessary dependencies and stuff.

Reply:
Yeah, probably. Not sure how to go about that for right now. But we should do it before merging. Maybe we can talk about this later?